jsk_apc2016_common/CMakeLists.txt

```diff
@@ -200,6 +200,14 @@ install(DIRECTORY include/sib/
   PATTERN ".svn" EXCLUDE
 )
 
+## rqt plugin
+install(FILES plugin.xml
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(PROGRAMS bin/target_select
```

Comment:
rqt_select_target, i prefer.
we use term 'work_order' for robot working order, it's time depend continuous line of target object and bin.
in this rqt, we just choose target, and after that, `work_order.py` decide which object robot should pick first.